src/styles/index.scss

```diff
@@ -118,4 +118,9 @@ body, html, .lg-container {
   }
 }
 
-
+small {
```

Comment:
Did you test if the styling still works on the results/webcam page?

Reply:
yes, it stil works